### PR TITLE
Closes #43 codeowners and #47 contributors

### LINKED ISSUE #43: Full list of CODEOWNERS
### Background Information

Programs | CODEOWNERS
-- | --
ae.R |  
cm.R |  
dm.R |  
ds.R |  
ex_ophtha.R |  
ex.R |  
lb.R |  
mh.R |  
oe_ophtha.R |  
pc.R |  
pp.R |  
qs_ophtha.R |  
qs.R |  
query_databases.R |  
rs_onco.R |  
sc_ophtha.R |  
sv.R |  
tr_onco.R |  
ts.R |  
tu_onco.R |  
vs.R |  

### Definition of Done

_No response_

### PR DESCRIPTION
**Pull request by kaz462**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -3,7 +3,12 @@ Type: Package
 Title: Test Data for the Pharmaverse Family of Packages
 Version: 0.0.2
 Authors@R: c(
-  person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = c("aut", "cre"))
+  person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = c("aut", "cre")),
+  person("Kangjie", "Zhang", role = "aut"),
+  person("Stefan", "Bundfuss", role = "aut"),
```
Comment (manciniedoardo) [inline DESCRIPTION:8]: please can you sort alphabetically below the maintainer, similar to what admiral and admiraldev are doing - thanks

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,5 +1,24 @@
 # This a CODEOWNERS file, where you can establish code owners.
 # Code owners are automatically requested for review when someone opens a pull request
-# that modifies code that they own. In this instance, cicdguy will be notified of any
-# changes made the files inside the .github/workflows directory.
-.github/workflows/* @cicdguy
+# that modifies code that they own.
+
+# admiral.test
+data-raw/ds.R @Gopi2345
+data-raw/lb.R @yangc71
```
Comment (manciniedoardo) [inline .github/CODEOWNERS:7]: do we need to give some of these people access to the repo? otherwise the codeowners file will have errors.

**Review by manciniedoardo — CHANGES_REQUESTED** [review]

**Comment by kaz462** [comment]
Hi @Gopi2345 @yangc71 @toni-1991 @RealSlimMahdi,
I found you through the history of the following data programs, can we list you as code owners for them? 
Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
Thanks!

data-raw/ds.R @Gopi2345           (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_ds.R)
data-raw/lb.R @yangc71     (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_lb.R)
data-raw/mh.R @yangc71   (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_mh.R)
data-raw/pc.R @toni-1991 @RealSlimMahdi   (from https://github.com/pharmaverse/admiral.test/blob/main/dev/pc.R)
data-raw/pp.R @toni-1991    (from https://github.com/pharmaverse/admiral.test/blob/main/dev/pp.R)

File: .github/CODEOWNERS (hunk 2 above)
Comment (kaz462) [reply .github/CODEOWNERS:7]: it seems so, could you please grant them access if they are okay to be listed as code owners? thanks!

**Comment by toni-1991** [comment]
> Hi @Gopi2345 @yangc71 @toni-1991 @RealSlimMahdi, I found you through the history of the following data programs, can we list you as code owners for them? Code owners are automatically requested for review when someone opens a pull request that modifies code that they own. Thanks!
> 
> data-raw/ds.R @Gopi2345 (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_ds.R) data-raw/lb.R @yangc71 (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_lb.R) data-raw/mh.R @yangc71 (from https://github.com/pharmaverse/admiral.test/blob/main/dev/update_mh.R) data-raw/pc.R @toni-1991 @RealSlimMahdi (from https://github.com/pharmaverse/admiral.test/blob/main/dev/pc.R) data-raw/pp.R @toni-1991 (from https://github.com/pharmaverse/admiral.test/blob/main/dev/pp.R)

I'm fine to be includedin PC and PP

**Comment by kaz462** [comment]
@toni-1991 Thanks a lot! 
@manciniedoardo could you please give @toni-1991 access to this repo?

**Review by manciniedoardo — APPROVED** [review]
